### ISSUE TITLE
Error in Go\Instrument\Transformer\FilterInjectorTransformer::rewrite when $resource is not a string

### ISSUE THREAD

**Issue by otereshchenkov** [goaop/framework#100]
Go\Instrument\Transformer\FilterInjectorTransformer::rewrite produces Fatal error if $resource parameter is not a string. For example:

``` php
$file = new SplFileInfo('example.php');
require_once $file;
```

This code is valid because SplFileInfo redeclares __toString() method and it returns path to file. This method is widely used in Symfony2:

``` php
// \Symfony\Bundle\FrameworkBundle\Test\WebTestCase::getKernelClass
$finder = new Finder();
$finder->name('*Kernel.php')->depth(0)->in($dir);
$results = iterator_to_array($finder);
$file = current($results);
require_once $file;
```

**Comment by lisachenko** [comment]
Thank you for the report. BTW, could you please provide a broken test for this case? PS. I haven't free time now for this project, so ping me if this issue is critical for you.

**Comment by otereshchenkov** [comment]
Looks like it's time to learn how to work with GitHub and how to make a pull requests. I've fixed this bug locally, so if I'll have a time at weekends, then I'll fix/add tests and create a pull request

**Comment by lisachenko** [comment]
Fixed in ea834ce3.
